lms/djangoapps/teams/api.py

```diff
@@ -0,0 +1,55 @@
+"""
+The Python API other app should use to work with Teams feature
+"""
+from __future__ import absolute_import
+
+from lms.djangoapps.teams.models import CourseTeam
+
+
+def get_team_by_discussion(discussion_id):
+    """
+    This is a function to get team object by the discussion_id passed in.
+    If the discussion_id is not associated with any team, we return None
+    """
+    try:
+        return CourseTeam.objects.get(discussion_topic_id=discussion_id)
+    except CourseTeam.DoesNotExist:
+        # When the discussion does not belong to a team. It's visible in
+        # any team context
+        return None
+
+
+def is_team_discussion_private(team, discussion_id):
```

Comment:
I think you can drop the `discussion_id` parameter -- every team has exactly one discussion, so it's redundant.